Makefile

```diff
@@ -23,26 +23,26 @@ FFLAGS =
 ifeq ($(FORTRAN),ifort)
 #  FFLAGS += -ftz -fp-model precise
   FFLAGS += -fp-model precise
-
+  
   # Work around for an incompatibility with some versions of ifort and OSX.
 #  FFLAGS += -use-asm
 
   # Debug options.
   FFLAGS += -g -O0 -traceback -fp-stack-check -check bounds -check uninit -fpe0 -ftrapuv
-
+  
   # Open/MP
-  FFLAGS += -qopenmp
+#  FFLAGS += -openmp
```

Comment:
Did you intent to change the openmp flag and comment it out?

Reply:
Line 34 is commented out in Makefile as I have it from Yunqian.  I'll uncomment it, build the code and test.

Reply:
There is no difference in the regression tests results as reported by the script run-regress.csh.

Reply:
I believe the flag is need for OpenMP test and it should be "-qopenmp" for the latest versions of the intel compiler.

Reply:
set this back to be added